### PR TITLE
[UTILS] new util tools related to FLASHDeconv

### PR DESCRIPTION
**Pull request by JeeH-K**
# Description

New util tools related to FLASHDeconv (Split from PR #5015)
To be merged after PR #5688

# Checklist:
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes. (Tick if no updates were necessary.)

# How can I get additional information on failed tests during CI:
If your PR is failing you can check out 
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. If you click in the column that lists the failed tests you will get detailed error messages.

# Note:
- Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

File: src/utils/AnalyzeFLASHIda.cpp [hunk 1]
```diff
@@ -0,0 +1,454 @@
...
+#include <OpenMS/METADATA/SpectrumLookup.h>
+using namespace OpenMS;
+using namespace std;
+
+//-------------------------------------------------------------
+// Doxygen docu
+//-------------------------------------------------------------
+
+/**
+    @page
```
Comment (timosachsenberg) [inline src/utils/AnalyzeFLASHIda.cpp:53]: please add documentation

File: src/utils/AnalyzeFLASHIda.cpp [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+public:
+  TOPPAnalyzeFLASHIda() :
+      TOPPBase("AnalyzeFLASHIda", ".", false)
+  {
+  }
+
+protected:
+  void registerOptionsAndFlags_() override
+  {
+    //registerOutputFileList_("out_spec", "<file for MS1, file for MS2, ...>", {""},
```
Comment (timosachsenberg) [inline src/utils/AnalyzeFLASHIda.cpp:70]: please remove commented out lines in files

File: src/utils/AnalyzeFLASHIda.cpp [hunk 3]
```diff
@@ -0,0 +1,454 @@
...
+    //setValidFormats_("i1n", ListUtils::create<String>("tsv"));
+    //setValidFormats_("i1n", ListUtils::create<String>("tsv"));
+  }
+
+  struct TopPicItem
+  {
+  public:
+    TopPicItem() = default;
+
+    TopPicItem(String in)
```
Comment (timosachsenberg) [inline src/utils/AnalyzeFLASHIda.cpp:85]: - mark single argument cosntructor with explicit
- can in be passed as const String& ?

File: src/utils/AnalyzeFLASHIda.cpp [hunk 4]
```diff
@@ -0,0 +1,454 @@
...
+        results.push_back(str);
+      }
+      //cout<<str_<<endl;
+      protein_acc_ = results[0];
+      proteform_id_ = stoi(results[1]);
+      rt_ = stod(results[2]);
+      precursor_mass_ = stod(results[4]);
+      precursor_mz_ = stod(results[5]);
+      precursor_intensity = stod(results[6]);
+        mass_intensity_ = stod(results[7]);
```
Comment (timosachsenberg) [inline src/utils/AnalyzeFLASHIda.cpp:103]: additional whitespace

File: src/utils/AnalyzeFLASHIda.cpp (hunk 2 above)
Comment (KyowonJeong) [reply src/utils/AnalyzeFLASHIda.cpp:70]: Oh I think I need to remove those files in utils.    I forgot those. I will remove those. Also as far as I remember I also need to remove those utils from the list of utils.

File: src/utils/AnalyzeFLASHIda.cpp [hunk 5]
```diff
@@ -0,0 +1,454 @@
...
+    double tol = 20.0;
+    fstream outstream;
+    outstream.open(outfile, fstream::out); //
+
+    MSExperiment map;
+    MzMLFile mzml;
+
+    mzml.setLogType(log_type_);
+    mzml.load(infiles[0], map);
+    auto iso_mzs = DoubleList{126.127725, 127.124760, 128.134433, 129.131468, 130.141141, 131.138176};
```
Comment (timosachsenberg) [inline src/utils/AnalyzeFLASHIda.cpp:140]: can you document those constants?

File: src/utils/AnalyzeFLASHIda.cpp (hunk 4 above)
Comment (KyowonJeong) [reply src/utils/AnalyzeFLASHIda.cpp:103]: Timo - I will remove these files - please ignore those in src/utils

File: src/utils/AnalyzeFLASHIda.cpp (hunk 5 above)
Comment (KyowonJeong) [reply src/utils/AnalyzeFLASHIda.cpp:140]: Timo - I will remove these files - please ignore those in src/utils Sorry for completely forgetting them.

File: src/utils/AnalyzeFLASHIda.cpp (hunk 1 above)
Comment (KyowonJeong) [reply src/utils/AnalyzeFLASHIda.cpp:53]: Most of these tools will be removed soon. Sorry for dirty codes.

**Pull request closed without merging**